metron-platform/metron-elasticsearch/pom.xml

```diff
@@ -206,24 +206,7 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
-        <dependency>
```

Comment:
Why were these dependencies removed?  Just curious.

Reply:
They cause the tests to fail when executed in an IDE like IntelliJ.  I don't understand exactly why, but @justinleet pointed me in this direction.

Also, everything runs just fine without them, so they are unnecessary.  The fewer dependencies, the better.

Reply:
I have also noticed this and I end up commenting these out every time I run a test in my IDE.  Thanks for investigating and removing them.